lib/main.dart

```diff
@@ -184,6 +188,17 @@ class _TenTenOneState extends State<TenTenOneApp> {
     }
   }
 
+  Future<void> _callGetOffers() async {
+    try {
+      // TODO: the endpoint should be configurable, probably in the settings?
```

Comment:
Let's make sure to cover this in a ticket after this gets merged. It's easy to overlook otherwise.

Reply:
I think we need this more globally because I am also working on adding calls to the maker's API.

I opened https://github.com/itchysats/10101/pull/209 for now to align this in once place.

Reply:
Here's what we came up with, with this you won't need to pass the endpoint via the API, but can just use it inside the rust code:
https://github.com/itchysats/10101/pull/210

Reply:
Extracted into a const in rust.

https://github.com/itchysats/10101/pull/197/commits/4c34e9813bfceeb710430ccbf3fd7780978eaa41#diff-45f0b65e906e22c7ca591e0d912bc35123341f4ea5c50e5207ae63b81d62b054R8